api-reference/docstubs/api/activitystatistics-list.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "List activityStatisticses"
+description: "List properties and relationships of the activityStatistics objects."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# List activityStatisticses
+
+Namespace: microsoft.graph
+
+List properties and relationships of the [activityStatistics](../resources/activitystatistics.md) objects.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /activitystatistics
```

Comment:
Interesting that this is not lowerCamelCased.  Is the entity set not declared properly?

Reply:
Correct: <EntitySet Name="activitystatistics" EntityType="microsoft.graph.activityStatistics"/>

Reply:
However, can either @wandeg or @dkershaw10 tell me why we generate this line:
GET /activitystatistics

This is the definition in the beta csdl:
`<EntityType Name="user" BaseType="graph.directoryObject" OpenType="true">
...
    <NavigationProperty Name="analytics" Type="graph.userAnalytics" ContainsTarget="true"/>
...
>
<EntityType Name="userAnalytics" BaseType="graph.entity">
    <Property Name="settings" Type="graph.settings"/>
    <NavigationProperty Name="activityStatistics" Type="Collection(graph.activityStatistics)" ContainsTarget="true"/>
</EntityType>`

The GET operation should be only user-based:
GET /me/analytics/activityStatistics
GET /users/{id}/analytics/activityStatistics

Reply:
> Interesting that this is not lowerCamelCased. Is the entity set not declared properly?

Yeah the casing comes directly from the type's name. This is an issue with the csdl not the code.

Reply:
@angelgolfer-ms at the moment the rules for the GET /type are not very clear. So what's happening here is we're generating the GET for each type that has an entityset.